frontend/src/types/tab.ts

```diff
@@ -1,12 +1,12 @@
 export interface TabInfo {
   id: string;
-  label: string;
+  name: string;
   isSaved: boolean;
   savedAt: string;
```

Comment:
It's unclear what's stored in savedAt is this a timestamp or url?

Reply:
timestamp

Reply:
To be consistent, it should be savedTs: number

https://github.com/bytebase/bytebase/blob/main/docs/api-style-guide.md#misc

Reply:
Let's follow up in a separate PR